docs/data/material/getting-started/learn/learn.md

```diff
@@ -1,61 +1,65 @@
-  - 📝 [The blog post](https://medium.com/codingthesmartway-com-blog/getting-started-with-material-ui-for-react-material-design-for-react-364b2688b555)
-  - 📹 [The video](https://www.youtube.com/watch?v=PWadEeOuv5o)
-- **Elegant UX in React with MUI**: a blog post covering some important MUI concepts.
-  - 📝 [The blog post](https://www.digitalocean.com/community/tutorials/react-material-ui)
+- [**Material UI v5 Crash Course**](https://www.youtube.com/watch?v=o1chMISeTC0) video by Laith Harb: everything you need to know to start building with the latest version of Material UI.
 
-### Paid
+- [**React + Material UI - From Zero to Hero**](https://www.youtube.com/playlist?list=PLDxCaNaYIuUlG5ZqoQzFE27CUOoQvOqnQ) video series by The Atypical Developer: build along with this in-depth series, from basic installation through advanced component implementation.
 
-- **Implement high fidelity designs**: Bridge the gap between Design & Development. Break down detailed designs and bring them to life with MUI and React.
+- [**Next.js 11 Setup with Material UI v5**](https://www.youtube.com/watch?v=IFaFFmPYyMI) by Leo Roese: learn how to integrate Material UI into your Next.js app, using Emotion as the style engine.
 
-  - 💻 [The Course](https://www.udemy.com/course/implement-high-fidelity-designs-with-material-ui-and-reactjs/)
+- [**Create a Responsive Navigation Bar with Material UI v5**](https://www.youtube.com/watch?v=lUkxSnJ7aDw) by Nikhil Thadani: detailed walkthrough using Material UI with Create React App.
 
-- **Apply Google Material Design**: This course teaches the fundamentals of Google Material Design and how to develop an end-to-end flight search and booking application using MUI and React.
+- [**The Clever Dev**](https://www.youtube.com/channel/UCb6AZy0_D1y661PMZck3jOw) and [**The Smart Devpreneur**](https://smartdevpreneur.com/category/javascript/material-ui/) by Jon M: dozens of high-quality videos and articles digging deep into the nuts and bolts of Material UI.
+
+### Paid
 
-  - 📹 [Watch Course Trailer](https://www.youtube.com/watch?v=hhZ6yFvCWho)
-  - 💻 [The Course](https://bonsaiilabs.com/courseDetail/material-ui-with-react/)
+- [**Implement High Fidelity Designs with Material-UI and ReactJS**](https://www.udemy.com/course/implement-high-fidelity-designs-with-material-ui-and-reactjs/) course by Zachary Reese: Bridge the gap between design & development. Break down detailed designs and bring them to life with Material UI (v4) and React.
 
-- **Cookbook**: Build modern-day applications by implementing Material Design principles in React, using MUI.
-  - 📘 [The book](https://www.amazon.com/gp/product/1789615224/)
+- [**Material UI with React**](https://bonsaiilabs.com/courseDetail/material-ui-with-react/) course by bonsaiilabs: Learn the fundamentals of Google Material Design and how to develop an end-to-end flight search and booking application using Material UI and React.
```

Comment:
This course never really took up, I would remove it, it's also on v4.

Reply:
Should we consider removing the Paid resources section entirely? The two remaining courses appear to be in the process of updating to v5, but I don't know when we can expect them to be released. We could remove the section for now and replace it when new content is available.

Reply:
Sam agree, let's comment out these section and add a comment to re-visit this in maybe Q3 and check if we should add those back in. We would also remove the Note we have above the free resources.